internal/common/config/config.go

```diff
@@ -62,13 +66,20 @@ type DisasterConfig struct {
 // (Made a function so it cannot be altered mid-game).
 func GameConfig() Config {
 	foragingConf := ForagingConfig{
+		//Deer parameters
 		MaxDeerPerHunt:        4,
 		IncrementalInputDecay: 0.8,
 		BernoulliProb:         0.95,
 		ExponentialRate:       1,
 
 		MaxDeerPopulation:     12,
 		DeerGrowthCoefficient: 0.4,
+
+		// Fish parameters
+		MaxFishPerHunt:            6,
+		FishIncrementalInputDecay: 0.8,
+		Mean:                      0.9,
+		Varience:                  0.2,
```

Comment:
I guess not--I think this PR is a good place to separate deer hunting and fishing parameters into different structs?

Reply:
can do, I just thought we were going to keep all foraging parameters within one struct

Reply:
Yeah I think unless we're going to preface those params with Fishing-, I think it becomes a little ambiguous and a separate struct would be better.

Also, typo in variance